tests/Parser/LineCheckerTest.php

```diff
@@ -91,4 +91,14 @@ public function testIsDefinitionListEnded(): void
         self::assertFalse($this->lineChecker->isDefinitionListEnded('Term', '    Definition'));
         self::assertFalse($this->lineChecker->isDefinitionListEnded('', '    Definition'));
     }
+
+    public function testIsFieldList(): void
+    {
+        self::assertTrue($this->lineChecker->isFieldList(':glob:'));
+        self::assertTrue($this->lineChecker->isFieldList(':Date: 2001-08-16'));
+        self::assertTrue($this->lineChecker->isFieldList('    :Date: 2001-08-16'));
+        self::assertTrue($this->lineChecker->isFieldList(':Date published: 2001-08-16'));
+        self::assertTrue($this->lineChecker->isFieldList(':Date:published: 2001-08-16'));
```

Comment:
> but colons (":") inside of field names must be backslash-escaped when followed by whitespace

That case does not seem to be tested. Also, the test determines whether there is a field list or not, but it does not say which part is the field name, and which is the body. Shouldn't there be a test for that?